src/CoreBundle/private/modules/CMSTemplateEngine/views/cmp_layoutlist.view.php

```diff
@@ -1,5 +1,19 @@
 <?php
 
+use ChameleonSystem\CoreBundle\ServiceLocator;
+use ChameleonSystem\CoreBundle\Util\UrlUtil;
+
+/**
+ * @var UrlUtil $urlUtil
+ */
+$urlUtil = ServiceLocator::get('chameleon_system_core.util.url');
+$editLanguage = ServiceLocator::get('chameleon_system_core.language_service')->getActiveEditLanguage();
+if (null === $editLanguage) {
+    $previewLanguageId = TCMSConfig::GetInstance()->fieldTranslationBaseLanguageId;
```

Comment:
please use `\ChameleonSystem\CoreBundle\Service\LanguageService::getCmsBaseLanguageId()`

Reply:
done

Reply:
I think there was a misunderstanding. I would keep the edit language and use the base language as fallback only.

Reply:
Sorry, took the wrong exit - fixed now :-)